core/commands/cid.go

```diff
@@ -85,6 +85,11 @@ The optional format string is a printf style format string:
 			if opts.newCodec != 0 && opts.newCodec != cid.DagProtobuf {
 				return fmt.Errorf("cannot convert to CIDv0 with any codec other than dag-pb")
 			}
+
+			if baseStr != "" {
+				return fmt.Errorf("cannot convert to CIDv0 with any multibase other than the implicit base58btc")
```

Comment:
If i pass `base58btc` it will error, while it is a valid base for CIDv0 – please fix + add test :pray:

Reply:
Ping @susarlanikhilesh